CentralHub.Api.Model/Measurement.cs

```diff
@@ -0,0 +1,38 @@
+using System.ComponentModel;
+
+namespace CentralHub.Api.Model;
+
+public sealed class Measurement
+{
+    private Protocol _protocol;
+
+    [Obsolete("Deserialization only")]
+    public Measurement()
+    {
+    }
+
+    public Measurement(Protocol type, string macAddress, int rssi)
+    {
+        Type = type;
+        MacAddress = macAddress;
+        RSSI = rssi;
+    }
+
+    public string MacAddress { get; set; }
+
+    public enum Protocol
+    {
+        wifi,
+        bluetooth
+    }
+    public Protocol Type
+    {
+        get => _protocol; set
```

Comment:
This is formatted very strangely
```suggestion
        get => _protocol;
        set
```

Reply:
Btw does this serialize / deserialize to json properly?

Reply:
i'm assuming it just serializes to an integer?
yes it seems to work fine